mantidproject/mantid
# Make I0MonitorIndex Optional

Description of work.

**To test:**
- Unit tests and System tests should pass
- Tim Charlton should be given the opportunity to test this out



Fixes #15075

Either link to [Release notes](http://www.mantidproject.org/Release_Notes_3.7) change-set or state "Does not need to be in the release notes."

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
